src/docs/index_dev.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Brainly Style Guide</title>
+  <meta name="description" content="Brainly Front-End Style Guide Documentation">
+
+  <link rel="icon" type="image/x-icon" href="img/favicon.ico">
+  <link rel="icon" sizes="192x192" href="img/favicon-hd.png">
+  <meta name="theme-color" content="#6ED6A0">
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js"></script>
+
+</head>
+<body>
+<div id="root"></div>
+<script src="/dist/bundle.js"></script>
```

Comment:
The bundle is not created on file system.
In dev mode the whole bunlde is compiled and served from server memory (for speed)